xml/geo_manage_i.xml

```diff
@@ -28,161 +28,152 @@
-     <para>
-      In this case, <literal>&ticket1;</literal> will be granted to the site
-      <literal>192.168.201.100</literal>. Without the <option>-s</option>
-      option, booth would automatically connect to the current site (the site
-      you are running the booth client on) and would request the
-      <command>grant</command> operation.
-     </para>
-     <para>
-      Before granting a ticket, the command executes a sanity check. If the
-      same ticket is already granted to another site, you are warned about that
-      and are prompted to revoke the ticket from the current site first.
+     <para> In this case, <literal>&ticket1;</literal> will be granted to
+      the site <literal>192.168.201.100</literal>. Without the
+       <option>-s</option> option, booth would automatically connect to the
+      current site (the site you are running the booth client on) and would
+      request the <command>grant</command> operation. </para>
+     <para> Before granting a ticket, the command executes a sanity check. If
+      the same ticket is already granted to another site, you are warned about
+      that and are prompted to revoke the ticket from the current site first.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
     <term>Revoking a Ticket From a Site</term>
     <listitem>
-<screen>&prompt.root;<command>booth</command> revoke &ticket1;
+     <screen>&prompt.root;<command>booth</command> revoke &ticket1;
 booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
-     <para>
-      Booth checks to which site the ticket is currently granted and requests
-      the <command>revoke</command> operation for <literal>&ticket1;</literal>.
-      The revoke operation will be executed immediately.
-     </para>
+     <para> Booth checks to which site the ticket is currently granted and
+      requests the <command>revoke</command> operation for
+       <literal>&ticket1;</literal>. The revoke operation will be executed
+      immediately. </para>
     </listitem>
    </varlistentry>
   </variablelist>
 
-  <para>
-   The <command>grant</command> and (under certain circumstances),
+  <para> The <command>grant</command> and (under certain circumstances),
    <command>revoke</command> operations may take a while to return a definite
    operation's outcome. The client waits for the result up to the ticket's
-   <varname>timeout</varname> value before it gives up waiting. If the
-   <option>-w</option> option was used, the client will wait indefinitely
+    <varname>timeout</varname> value before it gives up waiting. If the
+    <option>-w</option> option was used, the client will wait indefinitely
    instead. Find the exact status in the log files or with the
-   <command>crm_ticket -L</command> command.
-  </para>
+   <command>crm_ticket -L</command> command. </para>
+  <para> The <command>grant</command> command can take a <option>-F</option>
```

Comment:
Perhaps, worth mentioning here that the meaning of -F is different for automatic tickets?

Reply:
You mean ```booth grant -F```? Or do you mean ```crm_ticket --force```?

Reply:
"booth grant -F" I think. The behaviors on granting are described in "BOOTH TICKET MANAGEMENT" section from the manual, which is basically for automatics tickets, and "MANUAL TICKETS" section for manual ones of course. Hope they are helpful :-)


  @ChrisKowalczyk?

Reply:
@gao-yan , @ChrisKowalczyk: I studied both sections of the man page again and I must admit I don't see much difference with ```booth grant -F``` for manual vs. automatic tickets: if I got it right, in any case the ticket is granted to the new site immediately (without having to wait for  any timeouts or delays - ticket expire time or acquire-after time, which are only defined for automatic tickets, of course). Also, in both cases, it is the responsibility of the admin to make sure the same ticket is not granted to another site already (or that the site who currently has the ticket is really 'dead'). Nevertheless I will try to rephrase and add a separate entry for the ```-F``` option.

Reply:
@taroth21 , well the meaning is similar in both cases:

* For manual tickets -F means: always grant the ticket even it you know it is granted elsewhere.
* For automatic tickets -F means: grant tickets without waiting for confirmation from other sites. If the application knows that an automatic ticket is granted somewhere, it will not allow to grant it even with -F flag. So, the difference is that for split brain situation, from the app point of view, it may be not clear if an automatic ticket is granted somewhere else. This is why we use -F flag there - "trust me, the ticket is not granted anywhere. do not wait for responses from other, unreachable sites, just grant it"

Reply:
Thanks, @ChrisKowalczyk, this made it clearer to me. I tried to reflect this in the update of this snippet.